orchagent/dash/dashhaorch.cpp

```diff
@@ -0,0 +1,510 @@
+#include "dashhaorch.h"
+
+#include "orch.h"
+#include "sai.h"
+#include "saiextensions.h"
+#include "dashorch.h"
+#include "crmorch.h"
+#include "saihelper.h"
+
+#include "taskworker.h" 
+#include "pbutils.h"
+
+using namespace std;
+using namespace swss;
+
+extern sai_dash_ha_api_t* sai_dash_ha_api;
+extern sai_dash_eni_api_t* sai_dash_eni_api;
+extern sai_object_id_t gSwitchId;
+
+static std::map<dash::types::HaRole, std::string> ha_role_to_str_map = {
```

Comment:
Is it necessary to explicitly define these maps? Might be simpler to use `dash::types::HaRole_Name` and `dash::types::HaRole_Parse`

Reply:
addressed